src/Microsoft.AspNetCore.Razor/Tokenizer/Symbols/SymbolExtensions.cs

```diff
@@ -23,14 +37,21 @@ public static LocationTagged<string> GetContent(this SpanBuilder span, Func<IEnu
 
         public static LocationTagged<string> GetContent(this IEnumerable<ISymbol> symbols, SourceLocation spanStart)
```

Comment:
Assume you had another variant of this method but of `IList<ISymbol>` (like what you have to deal with above). How many references are there to this one? Mostly curious but it might be efficient.